.github/ISSUE_TEMPLATE/Platform _Epic_Template.md

```diff
@@ -0,0 +1,24 @@
+---
+Name: Epic template
+About: Template for creating  platform epics
+Title: "[EPIC]"
+Labels: epic platform
+
+---
+
+## Outcome/Objective ##
```

Comment:
Remove the pound signs at the end for all these headers, it's not needed and because you don't have a space in the last two headers it's not displaying correctly.